src/assets/token.ts

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { getLisk32AddressFromAddress } from '@liskhq/lisk-cryptography';
+
+import {
+	MODULE_NAME_TOKEN,
+	MODULE_NAME_DPOS,
+	ADDRESS_LEGACY_RESERVE,
+	TOKEN_ID_LSK_MAINCHAIN,
+	LOCAL_ID_LSK,
+	RADIX_HEX,
+	MODULE_NAME_LEGACY,
+	LOCAL_ID_LENGTH,
+} from '../constants';
+
+import {
+	AccountEntry,
+	LegacyAccountEntry,
+	SupplySubstoreEntry,
+	TokenStoreEntry,
+	UserSubstoreEntry,
+	ModuleResponse,
+	LockedBalance,
+} from '../types';
+
+const nextLexicographicalOrder = (currentID: string) =>
+	(parseInt(currentID, RADIX_HEX) + 1).toString(RADIX_HEX).padStart(LOCAL_ID_LENGTH, '0');
+
+export const getLockedBalances = async (account: AccountEntry): Promise<LockedBalance[]> => {
+	let amount = 0;
```

Comment:
`amount` should be of type `BigInt`. Please adjust the implementation accordingly.
```suggestion
	let amount: bigint = BigInt('0');
```